src/Symfony/Bundle/SecurityBundle/composer.json

```diff
@@ -17,25 +17,25 @@
     ],
     "require": {
         "php": ">=5.3.9",
-        "symfony/security": "~2.8|~3.0.0",
-        "symfony/security-acl": "~2.7|~3.0.0",
-        "symfony/http-kernel": "~2.7|~3.0.0",
+        "symfony/security": "~2.8|~3.0",
+        "symfony/security-acl": "~2.7|~3.0",
+        "symfony/http-kernel": "~2.7|~3.0",
         "symfony/polyfill-php70": "~1.0"
     },
     "require-dev": {
-        "symfony/browser-kit": "~2.4|~3.0.0",
-        "symfony/console": "~2.7|~3.0.0",
-        "symfony/css-selector": "^2.0.5|~3.0.0",
-        "symfony/dom-crawler": "^2.0.5|~3.0.0",
+        "symfony/browser-kit": "~2.4|~3.0",
+        "symfony/console": "~2.7|~3.0",
+        "symfony/css-selector": "^2.0.5|~3.0",
+        "symfony/dom-crawler": "^2.0.5|~3.0",
         "symfony/form": "^2.8.18",
         "symfony/framework-bundle": "^2.8.18",
-        "symfony/http-foundation": "~2.7|~3.0.0",
+        "symfony/http-foundation": "~2.7|~3.0",
         "symfony/twig-bundle": "~2.7|~3.1.0",
         "symfony/twig-bridge": "^2.7.4|~3.1.0",
```

Comment:
these two cannot be unlocked - at least tests fail if I do
this is related to the introduction of CLassExistenceResource in Config 3.2,
that cannot be used because here because DI ~2.8 is needed